prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -102,7 +102,7 @@ utils::check_required_vars "${requiredVars[@]}"
 
 function installCli() {
   export INSTALL_DIR=${TMP_DIR}
```

Comment:
you can remove this line. This was used by the Kyma-cli script for installation the binary to temporary directory.